apps/web/src/views/LimitOrders/components/LimitOrderTable/index.tsx

```diff
@@ -5,35 +5,31 @@ import useGelatoLimitOrdersHistory from '../../hooks/useGelatoLimitOrdersHistory
 
 import { ORDER_CATEGORY } from '../../types'
 
-import CompactLimitOrderTable from './CompactLimitOrderTable'
-import SpaciousLimitOrderTable from './SpaciousLimitOrderTable'
 import Navigation from './TableNavigation'
+import ExistingLimitOrderTable from './ExistingLimitOrderTable'
 
 const OrderTable: React.FC<React.PropsWithChildren<{ isCompact: boolean; orderCategory: ORDER_CATEGORY }>> = memo(
-  ({ orderCategory, isCompact }) => {
+  ({ orderCategory }) => {
     const orders = useGelatoLimitOrdersHistory(orderCategory)
 
     return (
       <Navigation data={orders} orderCategory={orderCategory}>
-        {({ paginatedData }) =>
-          isCompact ? (
-            <CompactLimitOrderTable orders={paginatedData} />
-          ) : (
-            <SpaciousLimitOrderTable orders={paginatedData} />
-          )
-        }
+        {({ paginatedData }) => <ExistingLimitOrderTable orders={paginatedData} />}
       </Navigation>
     )
   },
 )
 
 const LimitOrderTable: React.FC<React.PropsWithChildren<{ isCompact: boolean }>> = ({ isCompact }) => {
   const { t } = useTranslation()
-  const [activeTab, setIndex] = useState<ORDER_CATEGORY>(ORDER_CATEGORY.Open)
+  const [activeTab, setIndex] = useState<ORDER_CATEGORY>(ORDER_CATEGORY.Existing)
   const handleClick = useCallback((tabType: ORDER_CATEGORY) => setIndex(tabType), [])
   const tabMenuItems = useMemo(() => {
+    if (activeTab === ORDER_CATEGORY.Existing) {
+      return [t('Existing Orders')]
+    }
     return [t('Open Orders'), t('Expired Order'), t('Order History')]
```

Comment:
so there will still show the ('Open Orders'), t('Expired Order'), t('Order History') tabs? or just drop them

Reply:
when i click the tabMenu header, the activeTab will be 0, so then the tabMenuItems will render to the old three tabs, and i can't back to the 'Existing Orders' tab